docs/hsi.xml

```diff
@@ -0,0 +1,875 @@
+      </para>
+      <itemizedlist>
+        <listitem>
+          <para>
+            For HSI-1 this should be available for use by the OS or applications <emphasis>v1.5.0</emphasis>
+          </para>
+        </listitem>
+      </itemizedlist>
+      <note>
+        <para>
+          See also:
+          <itemizedlist>
+            <listitem>
+              <ulink url="https://en.wikipedia.org/wiki/Trusted_Platform_Module">
+                Wikipedia TPM Article
+              </ulink>
+            </listitem>
+          </itemizedlist>
+        </para>
+      </note>
+    </refsect3>
+
+    <refsect3 id="org.fwupd.hsi.Mei.ManufacturingMode">
+      <title>ME not in manufacturing mode</title>
+      <para>
+        There have been some unfortunate cases of the ME being distributed in
+        manufacturing mode.
+        In manufacturing mode many features from the ME can be interacted with
+        that decrease the platform’s security.
+      </para>
+      <itemizedlist>
+        <listitem>
+          <para>
+            For HSI-1 this should be unset <emphasis>v1.5.0</emphasis>
+          </para>
+        </listitem>
+      </itemizedlist>
+      <note>
+        <para>
+          See also:
+          <itemizedlist>
+            <listitem>
+              <ulink url="http://blog.ptsecurity.com/2018/10/intel-me-manufacturing-mode-macbook.html">
+                ME Manufacturing Mode: obscured dangers
+              </ulink>
+            </listitem>
+            <listitem>
+              <ulink url="https://www.intel.com/content/www/us/en/security-center/advisory/intel-sa-00086.html">
+                Intel security advisory SA-00086
+              </ulink>
+            </listitem>
+          </itemizedlist>
+        </para>
+      </note>
+    </refsect3>
+
+    <refsect3 id="org.fwupd.hsi.Mei.OverrideStrap">
+      <title>ME Flash Descriptor Override</title>
+      <para>
+        The Flash Descriptor Security Override Strap is not accessible to end
+        users on consumer boards and Intel stresses that this is for debugging only.
+      </para>
+      <itemizedlist>
+        <listitem>
+          <para>
+            For HSI-1 this should be unset <emphasis>v1.5.0</emphasis>
+          </para>
+        </listitem>
+      </itemizedlist>
+      <note>
+        <para>
+          See also:
+          <itemizedlist>
+            <listitem>
+              <ulink url="https://chromium.googlesource.com/chromiumos/third_party/flashrom/+/master/Documentation/mysteries_intel.txt">
+                Chromium documentation for Intel ME
+              </ulink>
+            </listitem>
+          </itemizedlist>
+        </para>
+      </note>
+    </refsect3>
+
+    <refsect3 id="org.fwupd.hsi.Mei.Version">
+      <title>CSME Version</title>
+      <para>
+        Converged Security and Manageability Engine is a standalone management
+        module that can manage and control all the local devices without the host
```

Comment:
That's a bit of an overreach.  It only controls very specific devices.

Reply:
How about ... `manage and control some local devices`

Reply:
Yes